ua/org.eclipse.tips.core/src/org/eclipse/tips/core/internal/FinalTip.java

```diff
@@ -31,17 +31,19 @@ public class FinalTip extends Tip implements IHtmlTip {
 	private static final String EH1 = "</h1>"; //$NON-NLS-1$
 	private static final String H1 = "<h1>"; //$NON-NLS-1$
 
+	private Date creationDate;
+
 	/**
```

Comment:
i vote to remove such useless comments "Constructor" on the fly

Reply:
ade919d7d02308c4a8bf6a086269963a94f47612